libs/hooks/open-telemetry/README.md

```diff
@@ -16,33 +18,52 @@ Confirm that the following peer dependencies are installed.
 $ npm install @openfeature/js-sdk @opentelemetry/api
 ```
 
+## Hooks
+
+### TracingHook
+
+This hook adds a [span event](https://opentelemetry.io/docs/concepts/signals/traces/#span-events) for each feature flag evaluation.
+
+### MetricsHook
+
+This hook performs metric collection by tapping into various hook stages. Below are the metrics are extracted by this hook:
+
+- `feature_flag.evaluation_requests_total`
+- `feature_flag.evaluation_success_total`
+- `feature_flag.evaluation_error_total`
+- `feature_flag.evaluation_active_count`
+
```

Comment:
Remark - these names were derived based on the Prometheus guide - https://prometheus.io/docs/practices/naming/